src/goRunTestCodelens.ts

```diff
@@ -9,29 +9,28 @@ import vscode = require('vscode');
 import path = require('path');
 import { CodeLensProvider, TextDocument, CancellationToken, CodeLens, Command } from 'vscode';
 import { getTestFunctions } from './goTest';
+import { getTestEnvVars } from './util';
 import { GoDocumentSymbolProvider } from './goOutline';
 
 export class GoRunTestCodeLensProvider implements CodeLensProvider {
 	private readonly debugConfig: any = {
 				'name': 'Launch',
 				'type': 'go',
 				'request': 'launch',
-				'mode': 'test',
-				'env': {
-					'GOPATH': process.env['GOPATH']
```

Comment:
am guessing you are removing this because you expect this to be passed in the testEnvFile?

For someone who has set the GOPATH in the settings, but doesn't use the `testEnvVars` or the new `testEnvFile` setting, the debug test codelens will not work if this is removed.